packages/icecast/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://icecast.org
+TERMUX_PKG_DESCRIPTION="Icecast is a streaming media (audio/video) server"
+TERMUX_PKG_VERSION=2.4.3
+TERMUX_PKG_SRCURL=http://downloads.xiph.org/releases/icecast/icecast-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_DEPENDS="libcurl, libgnutls, libogg, libvorbis, libxml2, libxslt, mime-support, openssl"
```

Comment:
`libvorbis` itself depends on `libogg`, so I don't think `libogg` is needed here.